service/orchestrator/audit_scope.go

```diff
@@ -184,6 +185,13 @@ func (svc *Service) RemoveAuditScope(ctx context.Context, req *orchestrator.Remo
 	}
 	go svc.informToeHooks(ctx, &orchestrator.AuditScopeChangeEvent{Type: orchestrator.AuditScopeChangeEvent_TYPE_AUDIT_SCOPE_REMOVED, AuditScope: auditScope}, nil)
 
+	if req.RemoveEvaluationResults {
```

Comment:
Should it also be possible to delete the associated assessment results and, if applicable, the evidence?

Reply:
No. The assessment result are independent from the audit scope.